tests/unittest_nodes.py

```diff
@@ -542,6 +542,19 @@ def test_str(self):
     def test_unicode(self):
         self._test("a")
 
+    @pytest.mark.skipif(
+        not PY38, reason="needs to be able to extract kind from ast.Constant"
```

Comment:
What do you think about: "kind attribute for ast.Constant was added in 3.8"

Reply:
yes, much better!